test/github-issues/1200/entity/A.ts

```diff
@@ -0,0 +1,26 @@
+import {Entity} from "../../../../src/decorator/entity/Entity";
+import {Column} from "../../../../src/decorator/columns/Column";
+import {PrimaryGeneratedColumn} from "../../../../src/decorator/columns/PrimaryGeneratedColumn";
+import {B} from "./B";
+import {C} from "./C";
+
+@Entity()
+export class A {
```

Comment:
sorry for being annoying but "A", "B", "C", "n" arent a good names, its hard to work with such meaningless names. Plus they will appear in the top of the suggestions lists in IDEs when you search something with "A" or "B" or "C" first-letter name.